w3d_translator/validate.py

```diff
@@ -0,0 +1,31 @@
+import os
+import sys
+import shutil
+
+
+# Ensure input folder is a directory
+# TODO: More dir validation?
+def validate_input(dir):
```

Comment:
should this check that an xml file is in the directory?

Reply:
Oh yeah maybe. Would have to do separate validation for `in_dir` when `--multiple` if this was the case

Reply:
`validate_project` now checks to see if there's an `.xml` file in the directory